modules/Bio/EnsEMBL/Production/Utils/CopyDatabase.pm

```diff
@@ -631,14 +675,51 @@ sub copy_mysql_files {
   return $copy_failed;
 } 
 
+# Dump the database in /nfs/nobackup/ensembl/ensprod/copy_service
+# Create database on target server
+# Load the database to the target server
+sub copy_mysql_dump {
+  my ($source_dbh,$target_dbh,$source_db,$target_db) = @_;
+  my $copy_failed=0;
+  my $file='/nfs/nobackup/ensembl/ensprod/copy_service/'.$source_db->{host}.'_'.$source_db->{dbname}.'_'.time().'.sql';
```

Comment:
It'd be a hassle to have the path as a parameter, but at the same time, it's rather buried in the code. Perhaps a halfway house would be to define a constant at the top of the file, so that it's a bit more obvious, and easier to change in the future if necessary?

Reply:
@james-monkeyshines: That make perfect sense. You are right, it's a hassle to add it as a parameter especially as it would only very rarely change. I've added the path at the start of the script.